Formula/pickle.rb

```diff
@@ -0,0 +1,17 @@
+class Pickle < Formula
+  desc "PHP Extension installer"
+  homepage "https://github.com/FriendsOfPHP/pickle"
+  url "https://github.com/FriendsOfPHP/pickle/releases/download/v0.6.0/pickle.phar"
+  sha256 "0bffcbff8368b85fcf30b098b901067c5cea2abc71c4c907c836dbb8c6f91f55"
+  license "BSD-3-Clause"
+
+  bottle :unneeded
+
```

Comment:
`uses_from_macos "php"`

Reply:
Apparently, the php binaries that are bundled with macOS does not work.

* On Catalina, `ext-zip` is missing which causes the command to fail (see FriendsOfPHP/pickle#195)
* On Mojave and High Sierra, the php binary is apparently too old: we need at least php 7.2.

Is there a way I can specify that I need a php binary from Homebrew (either `php` or `php@7.3` or `php@7.2`)?

Reply:
I've added `depends_on "php"` to make the tests pass, but the formula should work with any versioned php formula as well. I need a little help here.

Reply:
`depends_on "php"` it is then

Reply:
All right.